react.gradle

```diff
@@ -186,7 +186,7 @@ afterEvaluate {
                     def hermesFlags;
                     def hbcTempFile = file("${jsBundleFile}.hbc")
                     exec {
-                        if (targetName.toLowerCase().contains("release")) {
+                        if (!targetName.toLowerCase().contains("debug")) {
```

Comment:
I am not sure changing this logic for everyone won’t break the build somewhere (assuming that some people may have custom debug-oriented buildTypes). Maybe the best solution would be to introduce a lambda similar to `enableHermesForVariant` or flag like we have with `bundleIn$targetName`, and configure this logic from outside.

Reply:
Yes that probably is a better solution

Reply:
@grit96, were you going to make the change based on the suggestion?